faculty/clients/auth.py

```diff
@@ -104,3 +105,12 @@ def __call__(self, request):
         request.headers["Authorization"] = header_content
 
         return request
+
+
+class SherlockMLAuth(FacultyAuth):
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            "SherlockMLAuth has been renamed FacultyAuth - please update "
+            "your code to match the new class name."
+        )
+        super().__init__(*args, **kwargs)
```

Comment:
Also, this is not Python2 compatible, but it goes away if we drop `SherlockMLAuth`